test/testdrive/mzcompose.py

```diff
@@ -90,9 +97,23 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         fivetran_destination_files_path="/share/tmp",
     )
 
+    sysparams = args.system_param
+    if not args.system_param:
+        sysparams = []
+
+    additional_system_parameter_defaults = {}
+    for val in sysparams:
+        x = val[0].split("=")
```

Comment:
You only one to split once; it's legal for the parameter value to contain `=`.

```suggestion
        x = val[0].split("=", maxsplit=1)
```

Reply:
Done!